pynicotine/pluginsystem.py

```diff
@@ -739,42 +735,49 @@ def plugin_settings(self, plugin_name, plugin):
         except KeyError:
             log.add_debug("No stored settings found for %s", plugin.human_name)
 
-    def get_command_descriptions(self, command_interface, search_query=None):
+    def get_autocomplete_commands(self, command_interface):
 
-        command_groups = {}
+        completion_list = []
 
-        if command_interface == "chatroom":
-            command_list = self.chatroom_commands
+        for command, data in self.commands.get(command_interface).items():
+            completion_list.append(f"/{command} ")
 
-        elif command_interface == "private_chat":
-            command_list = self.private_chat_commands
+            if not data:
+                continue
```

Comment:
This condition gate is not required, because the following `for` will pass in zero loops if `data` is an empty `[]` list.